styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -106,6 +111,7 @@
   private static final int DEFAULT_POLL_PODS_INTERVAL_SECONDS = 60;
   static final String STYX_WORKFLOW_SA_ENV_VARIABLE = "GOOGLE_APPLICATION_CREDENTIALS";
   private static final String STYX_WORKFLOW_SA_ID_ANNOTATION = "styx-wf-sa";
+  private static final String STYX_WORKFLOW_SA_SECRET_EPOCH_ANNOTATION = "styx-wf-sa-epoch";
```

Comment:
Be consistent with the actual string? Either `STYX_WORKFLOW_SA_EPOCH_ANNOTATION` or `styx-wf-sa-secret-epoch`.